sa/sa.go

```diff
@@ -1375,11 +1378,38 @@ func (ssa *SQLStorageAuthority) NewAuthorizations2(ctx context.Context, req *sap
 		if err != nil {
 			return nil, err
 		}
-		err = ssa.dbMap.Insert(am)
+
+		// Each authz needs a (?,?...), in the VALUES block
+		qs.WriteString(fmt.Sprintf("(%s),", strings.TrimRight(strings.Repeat(" ?,", 11), ",")))
```

Comment:
This can be shortened up by using the `strings.Builder` as an `io.Writer`.
```suggestion
		// Each authz needs a (?,?...), in the VALUES block.
		fmt.Fprintf(&queryBuff, "(%s),", strings.TrimRight(strings.Repeat(" ?,", 11), ","))
```

Reply:
Nice. Gonna use `questionsBuf` though.